orm_lib/src/DbClientImpl.cc

```diff
@@ -436,12 +436,23 @@ DbConnectionPtr DbClientImpl::newConnection(trantor::EventLoop *loop)
         }
         // Reconnect after 1 second
         auto loop = closeConnPtr->loop();
-        loop->runAfter(1, [weakPtr, loop, closeConnPtr] {
+        trantor::TimerId reconnect_timer_id = loop->runEvery(
+            1, [weakPtr, loop, closeConnPtr, reconnect_timer_id] {
```

Comment:
I suspect this reconnect_timer_id cannot be captured correctly. Consider using a smart pointer.

Reply:
Thanks for suggestion, I will change this as soon as I finish testing.

I hope to finish today.

Reply:
I reworked the code to make it work in this scenario also:

1) Database down
2) Start drogon application
3) drogon tries to create an initial connection to the database.
   4a) behavior of the previous version and the code without changes from this PR: unexpected exit.
   4b) second version: If the initial connection was not established, there will be a new attempt to connect to the database after 1 second.